### DISCUSSION TITLE
Add support for `fts` command

### DISCUSSION THREAD

**Discussion by Ethan-Arrowood** [libuv/libuv#3781] [Ideas]
While working on https://github.com/nodejs/node/pull/41439 I discovered the [fts](https://man7.org/linux/man-pages/man3/fts.3.html) command that should allow us to do proper recursive file tree traversal. I'm going to try and land a pure JS solution first, but if we can get `fts` added to libuv, I believe we could then make the recursive readdir and recursive opendir operations as fast as possible!

**Comment by bnoordhuis** [comment]
fts is basically a wrapper around opendir/readdir/closedir, it doesn't do anything you can't already do with libuv.

It's also not universally available (ex: musl libc doesn't support it) and even when it's there, it's sometimes broken beyond repair, see https://sourceware.org/bugzilla/show_bug.cgi?id=11460.

**Reply by Ethan-Arrowood** [reply]
Good to know! I'll have to look into the implementation then and see if I can achieve file recursion with the existing libuv operations. Thank you!
